### ISSUE TITLE
store selected answers and justifications and scores in the backend

### ISSUE THREAD

**Issue by tillmo** [tillmo/two_tiered_quiz#39]
a user that returns to the quiz page should not start from scratch, but from his/her previous state
It suffices to store that data once the user presses "submit"

**Comment by tillmo** [comment]
For a quiz, there are three possible states:
* 0 questions answered: "start"
* all questions answered: "completed"
* otherwise: "continue"

**Comment by tillmo** [comment]
just some hints:

- in `backend/lqserver/QuizBoard/models.py`, you need to add a `Justification` field to class `Response`.
- the code for storing, for user u and quiz q, the response selecting answer a_i and justification j_i for question q_i (i=1..n) is like
``` 
qt = QuizTakers.objects.create(user=u, quiz=q, correct_answers=..., completed=... )
Response.objects.create(quiztaker=qt,question=q_1,answer=a_1,justification=j_1)
...
Response.objects.create(quiztaker=qt,question=q_n,answer=a_n,justification=j_n)
```
Please do not hesitate to ask any questions.

**Comment by tillmo** [comment]
fixed by #64
